virtual/README.md

```diff
@@ -8,18 +8,13 @@ Set up a virtual cluster with DeepOps. Useful for...
 
 ## Prerequisites
 
-Running DeepOps virtually assumes that the host machine's OS is Ubuntu 16.04 or greater. If this is
-not the case, the `bootstrap_virtual.sh` and `cluster_up.sh` scripts may be modified to work with
-a different OS.
+Running DeepOps virtually assumes that the host machine's OS is Ubuntu 16.04 or greater. DeepOps is also supported on CentOS 7.6 or greater. If this is not the case, the `bootstrap_virtual.sh` and `cluster_up.sh` scripts may be modified to work with a different OS.
```

Comment:
Can we say instead...

Approved OS:
- Ubuntu 16.04 (or greater)
- CentOS 7.6 (or greater)

Running DeepOps virtually assumes that the host machine's OS is an approved OS. If this is not the case, the `bootstrap_virtual.sh` and `cluster_up.sh` scripts may be modified to work with a different OS.